src/pages/en/guides/markdown-content.md

```diff
@@ -1,153 +1,73 @@
-export default {
-  markdownOptions: {
-    render: [
-      '@astrojs/markdown-remark',
-      {
-        remarkPlugins: [[autolinkHeadings, { behavior: 'prepend' }]],
-      },
-    ],
-  },
-};
-```
 
-### Syntax Highlighting
+```markdown
+---
+# Example: src/pages/index.md
+title: Hello, World
+---
+# Hi there!
 
-Astro comes with built-in support for [Prism](https://prismjs.com/) and [Shiki](https://shiki.matsu.io/). By default, Prism is enabled. You can modify this behavior by updating the `@astrojs/markdown-remark` options:
+This is your first markdown page. It probably isn't styled much, although
+Markdown does support **bold** and *italics.* 
 
-```js
-// astro.config.mjs
-export default {
-  markdownOptions: {
-    render: [
-      '@astrojs/markdown-remark',
-      {
-        // Pick a syntax highlighter. Can be 'prism' (default), 'shiki' or false to disable any highlighting.
-        syntaxHighlight: 'prism',
-        // If you are using shiki, here you can define a global theme and
-        // add custom languages.
-        shikiConfig: {
-          theme: 'github-dark',
-          langs: [],
-          wrap: false,
-        },
-      },
-    ],
-  },
-};
+To learn more about adding a layout to your page, read the next section on **Markdown Layouts.**
 ```
 
-You can read more about custom Shiki [themes](https://github.com/shikijs/shiki/blob/main/docs/themes.md#loading-theme) and [languages](https://github.com/shikijs/shiki/blob/main/docs/languages.md#supporting-your-own-languages-with-shiki).
-
-## Markdown Pages
-
-Astro treats any `.md` files inside of the `/src/pages` directory as pages. These files can contain frontmatter, but are otherwise processed as plain markdown files and do not support components. If you're looking to embed rich components in your markdown, take a look at the [Markdown Component](#astros-markdown-component) section.
-
-### Layouts
+### Markdown Layouts
 
-Markdown pages have a special frontmatter property for `layout`. This defines the relative path to an `.astro` component which should wrap your Markdown content, for example a [Layout](/en/core-concepts/layouts) component. All other frontmatter properties defined in your `.md` page will be exposed to the component as properties of the `content` prop. The rendered Markdown content is placed into the default `<slot />` element.
+Markdown pages have a special frontmatter property for `layout` that defines the relative path to an `.astro` [layout component](/en/core-concepts/layouts). This component will wrap your Markdown content, providing a page shell and any other included page template elements. 
 
 ```markdown
 ---
-# src/pages/index.md
 layout: ../layouts/BaseLayout.astro
-title: My cool page
-draft: false
 ---
-
-# Hello World!
 ```
 
+For a layout to work with a Markdown page:
+1. The page's frontmatter data must be defined as the `content` prop.
+2. The layout must have a default [`<slot />`](/en/guides/slots) for the Markdown content.
```

Comment:
I just did [a little test](https://stackblitz.com/edit/github-qmutuh?file=src/layouts/main.astro) and a layout will actually “work” without using `content` and without a `<slot/>`. Obviously it won’t render the Markdown, but it’s not an error.

Given that, I might suggest slightly different wording:

```suggestion
A layout receives data from the Markdown page in two ways:
1. A page's frontmatter data is available as the `content` prop.
2. The Markdown content will be rendered to the layout’s default [`<slot />`](/en/guides/slots).
```

I can imagine some scenarios where you might need _just_ `content` or _just_ a `<slot/>`, so this phrasing doesn’t scare you off doing that.

Reply:
We're occassionally torn between being *honest* and providing effective guard rails. I think your wording does both, but it feels like it's focusing more on "how this works" vs being something people should generally be sure to do when creating your first layout for a Markdown page. 

I do like a little bit of an implied threat of it not working (as a treat), since I think Fred's original intention here was: "This is what you gotta do to make sure your Layout will render a Markdown page." I actually softened the original wording from "a few rules to follow...."

So, I personally like the more imperative tone from before, but if I strive for better accuracy, and stick to a "rules to follow" take:

A typical layout for Markdown pages:
1. Defines the `content` prop to access the Markdown page's frontmatter data.
2. Includes a default [`<slot />`](/en/guides/slots) to indicate where the page's Markdown content should be rendered.

(I was considering saying one or both are optional, but I think that's the kind of thing people who are really curious, or who have very specific needs, can experiment with on their own. And we can keep this as a good practice for most to keep in mind when building typical pages.)

Thoughts @delucis ?

Reply:
This makes a lot of sense! The one thing I'd change would be to say

1. _Uses_ the `content` prop…

As opposed to “defines”. Technically I guess you are declaring or defining that variable but I think it could be confusing because you “define” the actual data in the page not the layout.

Reply:
Ooh, I can go one better!

A typical layout for Markdown pages includes:

1. the content prop to access the Markdown page's frontmatter data.
2. a default [<slot />](https://github.com/en/guides/slots) to indicate where the page's Markdown content should be rendered.

??

Reply:
I prefer the one with verbs!

Reply:
OK, then what about "Declares" instead of defines? I think I really do want to convey that you gotta "const" it there. How does it look/feel in situ?

A typical layout for Markdown pages:

1. Declares the content prop to access the Markdown page's frontmatter data.
2. Includes a default [`<slot />`](https://github.com/en/guides/slots) to indicate where the page's Markdown content should be rendered.

Reply:
OK, a fuzzy bear said "Uses" too... so I'm outvoted! I give in!

Reply:
Accesses? Although I still prefer “uses”. I was assuming the code example is what people actually look to for usage, so “OK, so I need to _use_ this prop. How? Aha, here’s the example, that is how.”

Reply:
After taking a small review, to break the impasse, I think the best one that has been proposed, would be Sarah's earlier proposal, this would be adequate of an explanation, personally I feel the less directive this part is, as it is informing the reader of a behavior of astro, it serves little but to add to their understanding. The directive tonality is best served in other parts of the page, 
https://github.com/withastro/docs/pull/202#discussion_r825113900

Reply:
Gotta go with my Bear here, to break the tie!  ;)

Any time you can common factor out, you probably have a winner.